nativelib/src/main/scala/scala/scalanative/runtime/monitor/ObjectMonitor.scala

```diff
@@ -42,6 +42,9 @@ private[monitor] class ObjectMonitor() {
    */
   @volatile private var enterQueue: WaiterNode = _
 
+  override def toString(): String =
+    s"ObjectMonitor(owner=$ownerThread),waiting=$waiting,recursion=$recursion)"
```

Comment:
Default toString don't return the parameter names but usually just `ClassName(param1,param2,param3)`